### PR TITLE
Bunch of tiny fixes

### PR DESCRIPTION
**Pull request by egonelbre**

### CODE REVIEW

File: transport/udp.go [hunk 1]
```diff
@@ -27,52 +26,41 @@ func ListenUDP(addr string) (*UDPListener, error) {
...
 func (u *UDPListener) Next() (packet *pb.Packet, source *net.UDPAddr, err error) {
-	buf := u.pool.Get().([]byte)
-	defer func() {
-		if buf != nil {
-			u.pool.Put(buf)
-		}
-	}()
-
-	n, source, err := u.conn.ReadFromUDP(buf)
+	var buf [10 * 1024]byte
```
Comment (egonelbre) [inline transport/udp.go:42]: Was there a benefit of having this buffer pooled?

File: client.go [hunk 2]
```diff
@@ -156,7 +157,12 @@ func (c *UDPClient) Run(ctx context.Context) {
 	c.init()
 
 	ticker := utils.NewJitteredTicker(c.FlushInterval)
-	defer ticker.Stop()
+	var background errgroup.Group
+	defer func() { _ = background.Wait() }()
+	background.Go(func() error {
+		ticker.Run(ctx)
+		return nil
+	})
```
Comment (jtolio) [inline client.go:165]: surely ticker.Stop could do this so we don't have to make every NewJitteredTicker usage do this dance.

File: transport/udp.go (hunk 1 above)
Comment (jtolio) [reply transport/udp.go:42]: it seemed more useful in an earlier refactor. i agree this is better

**Review by jtolio — APPROVED** [review]

File: client.go (hunk 2 above)
Comment (egonelbre) [reply client.go:165]: I'm wondering whether we can actually get rid of the background processing of ticker altogether.

File: transport/udp.go (hunk 1 above)
Comment (egonelbre) [reply transport/udp.go:42]: Note, if it's still necessary, we can use a single cached buffer for the udpclient rather than use a pool.

**Pull request merged**